src/docdb/nodes.ts

```diff
@@ -101,11 +99,57 @@ export class DocDBCollectionNode implements INode {
 		return await docs;
 	}
 
+	async getChildren(): Promise<INode[]> {
+		const collLink: string = this.getCollLink();
+		const parentNode = this;
+		const client = new DocumentClient(this.db.getEndpoint(), { masterKey: this.db.getPrimaryMasterKey() });
+		let documents = await this.listDocuments(collLink, client);
+		return documents.map(document => new DocDBDocumentNode(document.id, parentNode, document));
+	}
+
+	async listDocuments(collSelfLink, client): Promise<any> {
+		let documents = await client.readDocuments(collSelfLink);
+		return await new Promise<any[]>((resolve, reject) => {
+			documents.toArray((err, cols: Array<Object>) => err ? reject(err) : resolve(cols));
+		});
+	}
+
 	async readOneCollection(selfLink, client): Promise<any> {
 		let documents = await client.readDocuments(selfLink, { maxItemCount: 20 });
 		return await new Promise<any[]>((resolve, reject) => {
 			documents.toArray((err, docs: Array<Object>) => err ? reject(err) : resolve(docs));
 		});
 	}
 
+}
+
+export class DocDBDocumentNode implements INode {
+	data: IDocDBDocumentSpec;
```

Comment:
I don't like storing the data like this. We should dynamically retrieve the data every time the user clicks the collection. In addition, we shouldn't display '_rid', '_self', '_etag', etc. in result.json because we don't want the user to edit it

You don't have to fix in this PR, but please file an issue and fix this as highest priority after you merge the PR